one_fm/api/api.py

```diff
@@ -96,15 +96,18 @@ def rename_post(posts):
         except Exception as e:
             print(frappe.get_traceback())
 
-"""
-This function stores FCM Token  and Device OS in employee doctype 
-that is fetched from device/app end when user logs in.
-Params: Employee ID (Single employee ID), FCM Token and Device OS comes from the client side.
-It returns true or false based on the execution.
-eg:bench execute --kwargs "{'employee_id':'HR-EMP-00002','fcm_token':'cKg2nMfqwUpDinaWmUmVXp:APA91bHYAT2UxhesFtFHgweK8umoXm2LR9xyvi1L1OoefDRb3f1etOQbR3MZRFwL7Vwgc6hHrXO7urQWg3iy5SwTh_iE0lQP0Ny3O2nkcK_5Zo_UCQPMbZORF2Y5beejJbiHcQXA_x1C', 'device_os':'IOS'}" one_fm.api.api.store_fcm_token 
-"""
 @frappe.whitelist()
 def store_fcm_token(employee_id ,fcm_token,device_os):
+    """
+    This function stores FCM Token  and Device OS in employee doctype 
+    that is fetched from device/app end when user logs in.
+    
+    Params: Employee ID (Single employee ID), FCM Token and Device OS comes from the client side.
+    
+    It returns true or false based on the execution.
+    
+    eg:bench execute --kwargs "{'employee_id':'HR-EMP-00002','fcm_token':'f0_1sEWK7kksiegwCZ7dUm:APA91bHCFXcmXdI7AnI_37dbfjTz5uKf46_kvwIXgmtSoxGCbApo4zFETfbaWaEj8FpKXzJlwUS6CTTCfSX8WStuiFx1oPR4gtH3I46-jNbQkbhfvXI_lR3mKDl6e2ek2nB_5OFTfH9c', 'device_os':'ios'}" one_fm.api.api.store_fcm_token 

```

Comment:
There's no need to provide a bench execute guide here.

Reply:
@Ehtesham599 Done!